File: 3367-find-the-sum-of-encrypted-integers/3367-find-the-sum-of-encrypted-integers.cpp
class Solution {
public:
    int sumOfEncryptedInt(vector<int>& nums) {
        for(int i=0;i<nums.size();i++){
            string k=to_string(nums[i]);
            char m=*max_element(k.begin(),k.end());
            int l=k.size()-1;
            int r=1;
            while(l){
                r=r*10+1;
                l--;
            }
            nums[i]=(m-'0')*r;

        }
        int z=accumulate(nums.begin(),nums.end(),0);
        return z;
    }
};

File: 3367-find-the-sum-of-encrypted-integers/3367-find-the-sum-of-encrypted-integers.py
class Solution:
    def sumOfEncryptedInt(self, nums: List[int]) -> int:
        def cry(k):
            r=0
            m=0
            while(k):
                m=max(m,k%10)
                r=10*r+1
                k//=10
            return m*r
        for i in range(len(nums)):
            nums[i]=cry(nums[i])
        return sum(nums)
        